base_synchro/base_synchro_obj.py

```diff
@@ -20,88 +20,82 @@
+    name = fields.Char(string='Server name', size=64, required=True)
+    server_url = fields.Char(string='Server URL', size=64, required=True)
+    server_port = fields.Integer(string='Server Port', size=64, required=True, default=8069)
+    server_db = fields.Char(string='Server Database', size=64, required=True)
+    login = fields.Char(string='User Name', size=50, required=True)
+    password = fields.Char(string='Password', size=64, required=True)
+    obj_ids = fields.One2many('base.synchro.obj', 'server_id', string='Models', ondelete='cascade')
+
+class base_synchro_obj(models.Model):
     '''Class to store the operations done by wizard'''
     _name = "base.synchro.obj"
     _description = "Register Class"
-    _columns = {
-        'name':fields.char('Name', size=64, select=1, required=1),
-        'domain':fields.char('Domain', size=64, select=1, required=1),
-        'server_id':fields.many2one('base.synchro.server', 'Server', ondelete='cascade', select=1, required=1),
-        'model_id': fields.many2one('ir.model', 'Object to synchronize', required=True),
-        'action':fields.selection([('d', 'Download'), ('u', 'Upload'), ('b', 'Both')], 'Synchronisation direction', required=True),
-        'sequence': fields.integer('Sequence'),
-        'active': fields.boolean('Active'),
-        'synchronize_date':fields.datetime('Latest Synchronization', readonly=True),
-        'line_id':fields.one2many('base.synchro.obj.line', 'obj_id', 'Ids Affected', ondelete='cascade'),
-        'avoid_ids':fields.one2many('base.synchro.obj.avoid', 'obj_id', 'Fields Not Sync.'),
-    }
-    _defaults = {
-        'active': lambda *args: True,
-        'action': lambda *args: 'd',
-        'domain': lambda *args: '[]'
-    }
     _order = 'sequence'
+
+    name = fields.Char(string='Name', size=64, select=1, required=1)
+    domain = fields.Char(string='Domain', size=64, select=1, required=1,default='[]')
+    server_id = fields.Many2one('base.synchro.server', string='Server', ondelete='cascade', select=1, required=1)
+    model_id =  fields.Many2one('ir.model', string='Object to synchronize', required=True)
+    action = fields.Selection([('d', 'Download'), ('u', 'Upload'), ('b', 'Both')], string='Synchronisation direction', required=True, default='d')
+    sequence =  fields.Integer(string='Sequence')
+    active =  fields.Boolean(string='Active', default=True)
+    synchronize_date = fields.Datetime(string='Latest Synchronization', readonly=True)
+    line_id = fields.One2many('base.synchro.obj.line', 'obj_id', string='Ids Affected', ondelete='cascade')
+    avoid_ids = fields.One2many('base.synchro.obj.avoid', 'obj_id', string='Fields Not Sync.')
+
     #
     # Return a list of changes: [ (date, id) ]
     #
 
-    def get_ids(self, cr, uid, object, dt, domain=[], context=None):
-        return self._get_ids(cr, uid, object, dt, domain, context=context)
+    @api.model
+    def get_ids(self, object, dt, domain=[], action=None):

```

Comment:
Use of reserved built-in keywords should be avoided.
Ex: type, object, id, dict, etc